src/NeoSharp.Persistence.RocksDB/RocksDbContext.cs

```diff
@@ -14,10 +14,12 @@ public class RocksDbContext : IDbContext
         #endregion
 
         #region Constructor 
-        protected RocksDbContext(IRepositoryConfiguration config)
+
+        public RocksDbContext(IRepositoryConfiguration config)
```

Comment:
Without public, is not possible to run the application. Why this not fail on UnitTest?